tests/emailpassword/test_emailverify.py

```diff
@@ -950,10 +952,11 @@ async def test_the_generate_token_api_with_valid_input_and_then_remove_token(dri
     verify_token = await create_email_verification_token(user_id)
     await revoke_email_verification_token(user_id)
 
-    if verify_token.token is None:
-        raise Exception("Should never come here")
-    response = await verify_email_using_token(verify_token.token)
-    assert response.status == "EMAIL_VERIFICATION_INVALID_TOKEN_ERROR"
+    if isinstance(verify_token, CreateEmailVerificationTokenOkResult):
```

Comment:
You have changed the semantics of the test. Eariler, we would thrown an error if `if verify_token.token is None:`. Not if that's the case, `if isinstance(verify_token, CreateEmailVerificationTokenOkResult):` will be `false`, but you don't throw an error anymore.